src/components/Dashboard/ProjectActivity/ReservesEventElem.tsx

```diff
@@ -14,7 +14,18 @@ import { smallHeaderStyle } from '../styles'
 export default function ReservesEventElem({
   printReservesEvent,
 }: {
-  printReservesEvent: PrintReservesEvent | undefined
+  printReservesEvent:
```

Comment:
stricter type checking should hopefully make it easier to catch missing properties